mutagen/mp4/__init__.py

```diff
@@ -891,6 +892,119 @@ def to_line(key, value):
         return u"\n".join(values)
 
 
+class MP4Chapters(object):
+    """MP4Chapters()
+
+    MPEG-4 Chapter information.
+
+    Supports the 'moov.udta.chpl' box.
+
+    Behaves like a list of tuples where each tuple contains:
+        timestamp (`int`): position from the start of the file in seconds
+        title (`str`): title of the chapter
+
+    """
+
+    def __init__(self, *args, **kwargs):
+        self._timescale = None
+        self._duration = None
+        self._chapters = []
+        super(MP4Chapters, self).__init__()
+        if args or kwargs:
+            self.load(*args, **kwargs)
+
+    def __iter__(self):
+        iter(self._chapters)
+
+    def __delitem__(self, key):
+        self._chapters.__delattr__(key)
```

Comment:
why do you call `__delattr__` on a list?

Reply:
Probably copy pasted from elsewhere, dropped.